test_aodncore/pipeline/test_storage.py

```diff
@@ -325,11 +334,50 @@ def test_delete_file(self, mock_rm_f):
 
         netcdf_dest_path = os.path.join(file_storage_broker.prefix, netcdf_file.dest_path)
 
-        self.assertEqual(1, mock_rm_f.call_count)
-        mock_rm_f.assert_any_call(netcdf_dest_path)
+        self.assertEqual(1, mock_os_remove.call_count)
+        mock_os_remove.assert_any_call(netcdf_dest_path)
 
         self.assertTrue(netcdf_file.is_stored)
 
+    def test_delete_regexes(self):
+        with self.assertRaises(ValueError):
+            self.test_broker.delete_regexes([r''])
```

Comment:
Oops, I wasn't paying attention before hitting the Merge button. This test is failing! Can't quite see why yet...